tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "./jest.setup.ts",
     "./src/**/*",
     "./test/**/*",
-    "vite.config.ts"
+    "vite.config.ts",
+    "scripts"
```

Comment:
if you include it here, you should then exclude it there: https://github.com/image-js/image-js-typescript/blob/main/tsconfig.cjs.json#L8 (because we don't want to publish these scripts)